conftest.py

```diff
@@ -402,3 +389,7 @@ def check_skip_condition():
     import os
     if os.getenv("SKIP_MODULES") == "1":
         pytest.skip("Skipping tests due to global configuration.", allow_module_level=True)
+
+def mask_sensitive(config):
+    """Mask sensitive keys like 'password' before logging."""
+    return {k: (v if k != "password" else "***") for k, v in config.items()}
```

Comment:
```suggestion
    return {k: (v if "pass" in k.lower() else "***") for k, v in config.items()}
```
Suggestion for a slightly more aggressive masking effort - to adapt as you see fit :-)